src/ServicePulse.Host/vue/src/composables/serviceLicense.ts

```diff
@@ -12,19 +15,38 @@ const upgradeProtectionExpired =
 const trialExpiring =
   '<div class="license-warning"><strong>Non-production development license expiring</strong><div>Your non-production development license will expire soon. To continue using the Particular Service Platform you\'ll need to extend your license.</div><a href="http://particular.net/extend-your-trial?p=servicepulse" class="btn btn-license-warning"><i class="fa fa-external-link-alt"></i> Extend your license</a><a href="#/configuration" class="btn btn-license-warning-light">View license details</a></div>';
 
-export let license = reactive({
+interface License extends LicenseInfo {
+  licenseEdition: ComputedRef<string>;
+  formattedExpirationDate: ComputedRef<string>;
+  formattedUpgradeProtectionExpiration: ComputedRef<string>;
+  formattedInstanceName: ComputedRef<string>;
+}
+
+const emptyLicense: License = {
   edition: "",
-  licenseEdition: "",
-  expiration_date: undefined,
-  formattedExpirationDate: "",
-  upgrade_protection_expiration: undefined,
-  formattedUpgradeProtectionExpiration: "",
+  expiration_date: "",
+  upgrade_protection_expiration: "",
   license_type: "",
   instance_name: "",
-  formattedInstanceName: "",
   trial_license: true,
-  license_status: "Unavailable",
-});
+  registered_to: "",
+  status: "",
+  license_status: LicenseStatus.Unavailable,
+  licenseEdition: computed(() => {
```

Comment:
I decided to move the computed into the "empty/default" object, thoughts @PhilBastian ?